lib/std/valgrind.zig

```diff
@@ -80,6 +80,70 @@ pub const ClientRequest = extern enum {
     ChangeErrDisablement = 6145,
     VexInitForIri = 6401,
     InnerThreads = 6402,
+
+    CLEAN_MEMORY = 1212612608,
```

Comment:
This don't go here: they should be in valgrind/helgrind.zig.
Also use the function `valgrind.ToolBase`.

See `CallgrindClientRequest` for an example.